content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+---
+linktitle: Wrinkle Masks for Skin Materials
+title: "Wrinkle Mask Workflow"
+description: "Learn about how to drive the blending between wrinkle maps with morph targets."
+toc: true
+weight: 200
+---  
+
+This document describes the workflow for driving the blending of different wrinkle layers in the [Skin Material Type](https://github.com/o3de/o3de/blob/development/Gems/Atom/Feature/Common/Assets/Materials/Types/Skin.materialtype) using morph targets. It can be seen in action in the [Old World teaser](https://www.youtube.com/watch?v=ebr8t_mz8p8) that came out with O3DE's launch. Following this workflow will allow for the automatic blending between wrinkle layers driven by the active morph targets of the actor component.
+
+{{< important >}}
+This workflow represents a minimum viable product for animating wrinkle masks. There are a number of aspects that could use feedback on how to improve, such as how the normal maps and blend masks are blended, and the potential need for corrective blend masks. If you are using this feature, please reach out to [sig-graphics-audio on Discord](https://discord.com/channels/805939474655346758/816043793576886273) with your suggestions!
+{{< /important >}}
```

Comment:
I believe the rule is that you can only amend the Known Issues section of the upcoming release notes, not any previous ones. And it's okay for the known issues to report on features that were released in a prior version. That's because it's possible for issues to be found and called out after their release and in future versions.

Reply:
I've created a wrinkle mask discussion on github, and removed this section. https://github.com/o3de/o3de/discussions/15809